src/api/app/models/project_log_entry.rb

```diff
@@ -27,9 +27,8 @@ def self.create_from(payload, created_at, event_model_name)
     entry
   end
 
-  # Delete old entries
-  def self.clean_older_than(date)
-    where('datetime < ?', date).delete_all
+  def self.cleanup
+    where(event_type: [:build_fail, :build_success]).where('datetime < ?', Time.zone.yesterday.beginning_of_day).delete_all
```

Comment:
why is `beginning_of_day` needed? :thinking: `yesterday` returns an instance of the class `Date` which is already at the beginning of the day

Reply:
Yeah I guess it's the same...

```
ProjectLogEntry.where(event_type: [:build_fail, :build_success]).where('datetime > ?', Time.zone.yesterday).to_sql
=> "SELECT `project_log_entries`.* FROM `project_log_entries` WHERE `project_log_entries`.`event_type` IN ('build_fail', 'build_success') AND (datetime > '2018-12-06')"
irb(main):025:0> ProjectLogEntry.where(event_type: [:build_fail, :build_success]).where('datetime > ?', Time.zone.yesterday.beginning_of_day).to_sql
=> "SELECT `project_log_entries`.* FROM `project_log_entries` WHERE `project_log_entries`.`event_type` IN ('build_fail', 'build_success') AND (datetime > '2018-12-06 00:00:00')"
```

Reply:
yes :eyes: